src/options.rs

```diff
@@ -806,19 +806,20 @@ fn get_use_battery(args: &BottomArgs, config: &Config) -> bool {
     false
 }
 
-#[allow(unused_variables)]
+#[cfg(feature = "gpu")]
 fn get_enable_gpu(args: &BottomArgs, config: &Config) -> bool {
-    #[cfg(feature = "gpu")]
-    {
-        if args.gpu.enable_gpu {
-            return true;
-        } else if let Some(flags) = &config.flags {
-            if let Some(enable_gpu) = flags.enable_gpu {
-                return enable_gpu;
-            }
-        }
+    if args.gpu.enable_gpu {
```

Comment:
Hm, might want to change the arg (and the config) to `disable_gpu` instead, as if it's enabled by default `--enable_gpu` as a arg does nothing.